Automattic/woocommerce-payments
# chore: rename PRB constants to ECE

Fixes #

#### Changes proposed in this Pull Request









Dependent on https://github.com/Automattic/woocommerce-payments/pull/9739

I noticed there is a `Payment_Request_Button_States` file used for ECE.
Since "Payment Request Button" refers to the older implementation of GooglePay/ApplePay, I think it's safe now to rename these constants to "Express Checkout Element".

#### Testing instructions





This change affects both ECE and tokenized ECE, but you can just test it on the non-tokenized ECE (ensure you have "Enable Cart-Token implementation for ECE" disabled in the dev tools)
- As a merchant, ensure you have GooglePay/ApplePay enabled in the WooPayments settings
- As a customer, navigate to a product page for a physical product
- Add the product to the cart
- Go to the cart page
- Click on the GooglePay/ApplePay button
- Select the UK address (UK addresses are "tricky" because they provide semi-anonymized data)
- The selection should go though
- Place the order
- The order should be successful
- Proceed with another product, but use the Canadian address on the GooglePay dialog (the Canadian address will submit `Quebec` for the state value, which will trigger the `get_normalized_state_from_pr_states` method, being modified in this PR)
- Placing the order should be successful

-------------------


**Post merge**
